omnibot/routes/api.py

```diff
@@ -107,48 +107,48 @@ def slack_event():
         msg = 'No team_id in event.'
         logger.error(
             msg,
-            extra={'app': api_app_id},
+            extra={'bot_id': api_app_id},
```

Comment:
do we need to add this to keyword like you did last time? https://github.com/lyft/omnibot-private/blob/master/manifest.yaml#L100